### PR TITLE
mass find-replaces a common typo

### PR DESCRIPTION
**Pull request by PKPenguin321**
this should be fine as long as there are no icons with the word "defence" in them

### CODE REVIEW

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -53,3 +53,12 @@
 	burn_state = FIRE_PROOF
 	armor = list(melee = 15, bullet = 35, laser = 35, energy = 20, bomb = 35, bio = 35, rad = 35) //Not like anything ever hits the arms anyways.
 
+/obj/item/clothing/gloves/rags
```
Comment (PKPenguin321) [inline code/modules/clothing/gloves/miscellaneous.dm]: whoops

**Comment by Xhuis** [comment]
# ___YES___

**Comment by GunHog** [comment]
serveimage

**Comment by Incoming5643** [comment]
**Patriotic Screaming**

File: code/game/mecha/mecha_actions.dm [hunk 2]
```diff
@@ -163,25 +163,25 @@
...
-		chassis.defence_mode = forced_state
+		chassis.defense_mode = forced_state
 	else
-		chassis.defence_mode = !chassis.defence_mode
-	button_icon_state = "mech_defense_mode_[chassis.defence_mode ? "on" : "off"]"
-	if(chassis.defence_mode)
-		chassis.deflect_chance = chassis.defence_mode_deflect_chance
-		chassis.occupant_message("<span class='notice'>You enable [chassis] defence mode.</span>")
+		chassis.defense_mode = !chassis.defense_mode
+	button_icon_state = "mech_defense_mode_[chassis.defense_mode ? "on" : "off"]"
```
Comment (GunHog) [inline code/game/mecha/mecha_actions.dm]: Did you check the make sure that the Defense Mode button for Durands is not broken by this?

**Comment by Man-Shroom-** [comment]
G O D  B L E S S
O
D

B
L
E
S
S

File: code/game/mecha/mecha_actions.dm (hunk 2 above)
Comment (PKPenguin321) [inline code/game/mecha/mecha_actions.dm]: nope

Comment (PKPenguin321) [inline code/game/mecha/mecha_actions.dm]: if you'd happen to know what file these icons are in hit me up

Comment (GunHog) [inline code/game/mecha/mecha_actions.dm]: `actions.dmi`. Having taken a look at your code and the sprite, it is already properly **AMERICAN** and should be entirely safe. You should still test, though.

**Comment by Cheridan** [comment]
these tags don't run

File: code/game/mecha/mecha_actions.dm (hunk 2 above)
Comment (ChangelingRain) [inline code/game/mecha/mecha_actions.dm]: You don't need to, the only change is to the `[ ? "":""]`

Comment (PKPenguin321) [inline code/game/mecha/mecha_actions.dm]: wait, so does it work or not

Comment (PKPenguin321) [inline code/game/mecha/mecha_actions.dm]: well i tested it and the button for the defense mode does indeed still have an icon and work so as far as i can tell there is no issue here

File: code/game/mecha/mecha_topic.dm [hunk 3]
```diff
@@ -86,7 +86,7 @@
 						<b>Cabin temperature: </b> [return_temperature()]&deg;K|[return_temperature() - T0C]&deg;C<br>
 						[dna_lock?"<b>DNA-locked:</b><br> <span style='font-size:10px;letter-spacing:-1px;'>[dna_lock]</span> \[<a href='?src=\ref[src];reset_dna=1'>Reset</a>\]<br>":""]<br>
 						[thrusters_action.owner ? "<b>Thrusters: </b> [thrusters_active ? "Enabled" : "Disabled"]<br>" : ""]
-						[defense_action.owner ? "<b>Defence Mode: </b> [defence_mode ? "Enabled" : "Disabled"]<br>" : ""]
+						[defense_action.owner ? "<b>defense Mode: </b> [defense_mode ? "Enabled" : "Disabled"]<br>" : ""]
```
Comment (phil235) [inline code/game/mecha/mecha_topic.dm]: Capital D.

Comment (PKPenguin321) [inline code/game/mecha/mecha_topic.dm]: Good catch

**Comment by optimumtact** [comment]
https://github.com/tgstation/tgstation/blob/master/CONTRIBUTING.md#enforced-not-enforced

**Comment by PKPenguin321** [comment]
woah what?

**Comment by PKPenguin321** [comment]
oh shit okay

i had to google it but apparently "defence" is actually how you spell it in europe/etc

i thought it was just straight up a misspelling my bad

**Comment by Man-Shroom-** [comment]
FUCK YOU ORANGE MAN YOU DIRTY COMMIE

**Comment by ike709** [comment]
Well it's actually spelled "defence" in the Constitution, which in my opinion makes it more American than "defense".

**Comment by RemieRichards** [comment]
Wasn't the Constitution written before Webster through a hissy fit and decided to create your version of the language?

**Comment by ike709** [comment]
@RemieRichards Shhh. :us: :us: :us:

**Pull request closed without merging**